cmake/android/android_gradle_projects.cmake

```diff
@@ -82,6 +82,13 @@ foreach(fname ${GRADLE_WRAPPER_FILES})
   install(FILES "${OpenCV_SOURCE_DIR}/platforms/android/gradle-wrapper/${fname}" DESTINATION "${ANDROID_INSTALL_SAMPLES_DIR}/${__dir}" COMPONENT samples ${__permissions})
 endforeach()
 
+# set build.gradle namespace 
+if(GRADLE_VERSION STREQUAL "5.6.4")
+  ocv_update(OPENCV_NAMESPACE_DECLARATION "")
+else()
+  ocv_update(OPENCV_NAMESPACE_DECLARATION "namespace 'org.opencv'")
+endif()
```

Comment:
What would happen if `GRADLE_VERSION` is 5.6.3?

Reply:
Namescape came with AGP 7.3 (https://developer.android.com/build/releases/gradle-plugin) so is it okay to set minimum 7.3.0 and any higher should be fine. Any lower version basically won't use it. That include 5.6.3 or any X version lower then 7.3.0

```
# set build.gradle namespace 
if(GRADLE_VERSION VERSION VERSION_GREATER_EQUAL "7.3.0")
  ocv_update(OPENCV_NAMESPACE_DECLARATION "namespace 'org.opencv'")
else()
  ocv_update(OPENCV_NAMESPACE_DECLARATION "")
endif()

```

Reply:
Right, condition should use GE version check instead of strong equality.

Reply:
Did you mean this ? 

if(GRADLE_VERSION VERSION VERSION_GREATER "7.2.9").

Reply:
Both cases should do the similar thing:

```
if(GRADLE_VERSION VERSION_GREATER_EQUAL "7.3.0")
if(GRADLE_VERSION VERSION_GREATER "7.2.9")
```

The first option is clear however.

---

BTW, "gradle plugin version" is not the same as "gradle version".
Plugin version is specified through `ANDROID_GRADLE_PLUGIN_VERSION`

Reply:
I changed with AGP and push it.